src/index.html

```diff
@@ -1,13 +1,292 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/styles/style.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <!-- #region header + menu -->
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#"class="top-bar__logo-link">
+              <img
+              class="top-bar__logo"
+              src="../src/images/logo-white-2x.png"
+              alt="MyBIKE logo">
+            </a>
+            <div class="top-bar__icons">
+              <a href="#" class="icon icon--phone"></a>
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+          <div class="header__bottom">
+            <h1 class="header__title">Take the Streets</h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo-link">
+            <img
+            class="top-bar__logo"
+            src="../src/images/logo-black-2x.png"
+            alt="MyBIKE logo">
+          </a>
+          <div class="header__icons">
+            <a href="#" class="icon icon--close"></a>
+          </div>
+        </div>
+
+        <nav class="nav menu__nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link" href="#">Home</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#move-free">About us</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#compare-bikes">Compare bikes</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#details">Details</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#contact-us">Contacts</a>
+            </li>
+          </ul>
+        </nav>
+
+        <a href="tel:+1 234 5555-55-55" class="menu__phone-number">
+        +1 234 5555-55-55
+        </a>
+
+        <a href="tel:+1 234 5555-55-55" class="menu__call-to-order">
+          Call to order
+        </a>
+      </div>
+
+    </aside>
+    <!-- #endregion header + menu -->
+    <main class="main">
+      <div class="container">
+        <div class="main__content">
+          <!-- #region move free -->
+          <section class="move-free" id="#move-free">
+            <h2 class="
+            move-free__title
+            section-title
+            section-title--align--left">
+            Move Free
+            </h2>
+            <p class="move-free--text">
```

Comment:
[using-a-modifier-without-the-belonging-class](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en#using-a-modifier-without-the-belonging-class)